continuous_integration/recipes/distributed/meta.yaml

```
@@ -18,6 +18,7 @@ build:
   noarch: python
   script: python -m pip install . -vv --no-deps
```

Based on [this line from the feedstock]( https://github.com/conda-forge/distributed-feedstock/blob/c69dfd4b07dda15f51b6aa18bb8826a2d7e72341/recipe/meta.yaml#L15 )

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```